apps/studio/components/interfaces/QueryPerformanceV2/QueryPanel.tsx

```diff
@@ -0,0 +1,75 @@
+import { InformationCircleIcon } from '@heroicons/react/16/solid'
+import { ArrowDown, ArrowUp } from 'lucide-react'
+import { PropsWithChildren } from 'react'
+import { TooltipContent_Shadcn_, TooltipTrigger_Shadcn_, Tooltip_Shadcn_, cn } from 'ui'
+
+export const QueryPanelContainer = ({
+  children,
+  className,
+}: PropsWithChildren<{ className?: string }>) => (
+  <div className={cn('flex flex-col gap-y-6 py-4', className)}>{children}</div>
+)
+
+export const QueryPanelSection = ({
+  children,
+  className,
+}: PropsWithChildren<{ className?: string }>) => (
+  <div className={cn('px-5 flex flex-col gap-y-2', className)}>{children}</div>
+)
+
+export const QueryPanelScoreSection = ({
+  className,
+  name,
+  description,
+  before,
+  after,
+}: {
+  className?: string
+  name: string
+  description: string
+  before?: number
+  after?: number
+}) => (
+  <div className={cn('flex flex-col gap-y-1', className)}>
+    <div className="flex items-center gap-x-2">
+      <span>{name}</span>
+      <Tooltip_Shadcn_>
+        <TooltipTrigger_Shadcn_ asChild>
+          <InformationCircleIcon className="transition text-foreground-muted w-3 h-3 data-[state=delayed-open]:text-foreground-light" />
+        </TooltipTrigger_Shadcn_>
+        <TooltipContent_Shadcn_ side="bottom" className="w-52 text-center">
```

Comment:
What do you think of making this top as well? That way it doesn't cover the numbers

<img width="372" alt="Screenshot 2024-04-11 at 15 31 52" src="https://github.com/supabase/supabase/assets/10985857/6ccb3bf1-0771-4a41-9a95-a9f728ddd591">

```suggestion
        <TooltipContent_Shadcn_ side="top" className="w-52 text-center">
```